barretenberg/cpp/src/barretenberg/bb/main.cpp

```diff
@@ -589,7 +590,7 @@ void prove_tube(const std::string& output_path)
 
     builder->add_recursive_proof(current_aggregation_object);
 
-    info("num gates in tube circuit: ", builder->get_num_gates());
+    info("num gates in tube circuit: ", builder->num_gates);
```

Comment:
maybe say that this is num prefinalized gates in tube circuit

Reply:
will remove